src/integration/common/renovate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": ["config:base", ":pinAllExceptPeerDependencies"],
```

Comment:
going for the default config for a js app https://docs.renovatebot.com/presets-config/#configjs-app

the config that makes up `config:base` can be viewed here https://docs.renovatebot.com/presets-config/#configbase

all configuration options can be viewed here https://docs.renovatebot.com/configuration-options/